packages/fluentui/react/src/themes/teams/components/Dropdown/dropdownSearchInputStyles.ts

```diff
@@ -1,17 +1,22 @@
 import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
 import { DropdownSearchInputProps } from '../../../../components/Dropdown/DropdownSearchInput';
 import { DropdownVariables } from './dropdownVariables';
+import getBorderFocusStyles from '../../getBorderFocusStyles';
 
 const dropdownSearchInputStyles: ComponentSlotStylesPrepared<DropdownSearchInputProps, DropdownVariables> = {
   root: ({ variables: v }): ICSSInJSStyle => ({
     flexBasis: v.comboboxFlexBasis,
     flexGrow: 1
   }),
 
-  input: ({ props: p }): ICSSInJSStyle => ({
+  input: ({ props: p, theme: { siteVariables } }): ICSSInJSStyle => ({
     width: '100%',
     backgroundColor: 'transparent',
     borderWidth: 0,
+    ':focus-within': {
```

Comment:
FYI: This will not work in IE and probably old Edge

Reply:
It was extra anyway. Removing it.